### ISSUE TITLE
Add ability to change default layer image

### ISSUE THREAD

**Issue by chuckdee68** [chuckdee68/IEDLCDEditor#2]
Per issue filed [here](https://geekhack.org/index.php?topic=82904.msg2213582#msg2213582), the default layer image cannot currently be changed, i.e.

> If you mean the default image color, ie. the gray logo color, then yes that's unchangeable in the current state of the editor.

**Comment by LuuuuX** [comment]
Yes, the default layer color seems to be hidden, or maybe stored somewhere else. The easiest way to find it would be to use the original https://github.com/kiibohd/controller to build two versions; one unaltered and one with the default color changed so something obvious. Then use a program like HxD to compare the two to find how and what has changed. Unfortunately I couldn't get the building to work on my system so I cannot do that personally.

From there adding that feature should be easy enough.

**Comment by hymnis** [comment]
The default image used for layer 0 can be found in: [scancode_map.kll](https://github.com/kiibohd/controller/blob/master/Scan/Infinity_Ergodox/scancode_map.kll), line 147 (STLcdDefaultImage). There are also default settings for the backlight in the same file.

I have been playing around with the code a bit and build some custom firmware for my ErgoDox. So I might try what you're suggesting LuuuuX and I'll post the results when I have.
